src/OrleansAWSUtils/AWSUtilsServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Orleans;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.MembershipService;
+using OrleansAWSUtils.Configuration;
+
+namespace Microsoft.Orleans.Hosting
```

Comment:
I think we were using `Orleans.Runtime.Hosting` as the namespace for the extension methods and the Options, don't recall exactly, but I don't think we were using Microsoft.

Reply:
I'd use Microsoft.Orleans.Hosting as the namespace for all these extension methods. Thoughts? /cc @ReubenBond @attilah

quote from you ;) wink wink. But  yeah I should know anything start with Microsoft.Orleans doesn't sound right.

Reply:
hehe, my mistake then, but I really meant `Orleans.Hosting` (where the `ISiloHostBuilder` is, which I'm now checking, it doesn't include Runtime either) instead of `OrleansAWSUtils` or some non easily  discoverable namespace when configuring the builder